src/Sentry.Android.AssemblyReader/V2/AndroidAssemblyStoreReaderV2.cs

```diff
@@ -119,10 +127,12 @@ private bool FindBestAssembly(string name, out ExplorerStoreItem? explorerAssemb
         {
             if (explorer.AssembliesByName?.TryGetValue(name, out var assembly) is true)
             {
+                _logger?.Invoke("Found best assembly {0} in APK AssemblyStore for target arch {1}", name, explorer.TargetArch);
                 explorerAssembly = new(explorer, assembly);
                 return true;
             }
         }
+        _logger?.Invoke("No best assembly for {0} in APK AssemblyStore", name);
```

Comment:
looks like we have one logger but shouldn't this be a level Error log?

Reply:
We'd have to refactor the AssemblyReader classes that we vendored in fomr MS in order to be able to do that (they use a different pattern for logging). 

Probably the easiest would be to replace [DebugLogger](https://github.com/getsentry/sentry-dotnet/blob/4ee96eeaf000d8f1d426a914622548a473616bd3/src/Sentry.Android.AssemblyReader/DebugLogger.cs#L8) with our own `IDiagnosticLogger`...  would involve quite a few changes, but then we've made so many changes to these already that the only way to merge in changes from new code is by hand. So a few more changes wouldn't hurt.

Reply:
@bruno-garcia one remaining question on this PR is whether we want to refactor all the vendored in code to use our  `IDiagnosticLogger` instead of the `DebugLogger` delegate that is used currently: 
https://github.com/getsentry/sentry-dotnet/blob/4ee96eeaf000d8f1d426a914622548a473616bd3/src/Sentry.Android.AssemblyReader/DebugLogger.cs#L8

The existing delegate doesn't have any notion of "log levels" (so doesn't support what you suggested above).

Reply:
k, I'll track that in a separate issue as I think we should get this fix out ASAP:
- https://github.com/getsentry/sentry-dotnet/issues/4382